users/tests/test_api_user.py

```diff
@@ -0,0 +1,222 @@
+        "banner": user.profile.banner,
+        "user": user.id,
+    }
+
+
+@pytest.mark.django_db()
+def test_patch_api_user_profile_valid_data_response_success(
+    authenticated_client: tuple[APIClient, str, str], api_user_profile_url: str, user: User
+) -> None:
+    client, _, _ = authenticated_client
+    data = {"is_followable": not user.profile.is_followable}
+    response = client.patch(api_user_profile_url, data=data)
+    assert response.status_code == 200
+    assert response.data == {
+        "id": user.profile.id,
+        "bio": user.profile.bio,
+        "is_nsfw": user.profile.is_nsfw,
+        "is_followable": not user.profile.is_followable,
+        "is_content_visible": user.profile.is_content_visible,
+        "is_communities_visible": user.profile.is_communities_visible,
+        "comment_karma": user.profile.comment_karma,
+        "post_karma": user.profile.post_karma,
+        "gold_awards": user.profile.gold_awards,
+        "gender": user.profile.gender,
+        "avatar": user.profile.avatar,
+        "banner": user.profile.banner,
+        "user": user.id,
+    }
+
+
+@pytest.mark.django_db()
+def test_patch_api_user_profile_invalid_data_response_failed(
+    authenticated_client: tuple[APIClient, str, str], api_user_profile_url: str, user: User
+) -> None:
+    client, _, _ = authenticated_client
+    data = {"is_followable": "TEST"}
+    response = client.patch(api_user_profile_url, data=data)
+    assert response.status_code == 400
+
+
+@pytest.mark.django_db()
+def test_api_user_profile_unauthenticated_user_response_failed(
+    client: Client, user_credentials: tuple[APIClient, str, str], api_user_profile_url: str
+) -> None:
+    response = client.get(api_user_profile_url)
+    assert response.status_code == 401
+    assert isinstance(response.data["detail"], ErrorDetail)
+
+
+@pytest.mark.django_db()
+def test_api_user_settings_valid_data_response_success(
+    authenticated_client: tuple[APIClient, str, str], api_user_settings_url: str, user: User
+) -> None:
+    client, _, _ = authenticated_client
+    response = client.get(api_user_settings_url)
+    assert response.status_code == 200
+    assert response.data == {
+        "id": user.usersettings.id,
+        "content_lang": user.usersettings.content_lang,
+        "location": user.usersettings.location,
+        "is_beta": user.usersettings.is_beta,
+        "revert_to_old_reddit": user.usersettings.revert_to_old_reddit,
+        "is_over_18": user.usersettings.is_over_18,
+        "user": user.id,
+    }
+
+
+@pytest.mark.django_db()
+def test_patch_api_user_settings_valid_data_response_success(
+    authenticated_client: tuple[APIClient, str, str], api_user_settings_url: str, user: User
+) -> None:
+    client, _, _ = authenticated_client
+    data = {"is_beta": not user.usersettings.is_beta}
+    response = client.patch(api_user_settings_url, data=data)
+    assert response.status_code == 200
+    assert response.data["is_beta"]
```

Comment:
Hmm actually I don't see a change here. It has some value, it's not showing that the call changed anything. Please prove that is_beta was changed.